### PR TITLE
Add gedit integration

### PR DESCRIPTION
**Pull request by Architrixs**
I don't know if anyone else need this, I find it useful.

### CODE REVIEW

File: docs/editor_integration.md [hunk 1]
```diff
@@ -233,6 +233,32 @@ If you later want to update _Black_, you should do it like this:
...
+```
+1. `Go to edit > preferences > plugins`
+2.  Search for `external tools` and activate it.
+3.  In `Tools menu -> Manage external tools`
+4.  Add a new tool using `+` button.
+5.  Copy the below content to the code window.
+```console
+#!/bin/bash
+Name = $GEDIT_CURRENT_DOCUMENT_NAME
+black $Name
```
Comment (ichard26) [inline docs/editor_integration.md:250]: I followed your steps (which are easy to follow btw!) and everything worked except for the actual bash script :(
Screenshot from 2021-02-03 19-06-23

Skipping the "saving to a variable" step and directly providing the document path to black worked tho.

File: docs/editor_integration.md [hunk 2]
```diff
@@ -233,6 +233,32 @@ If you later want to update _Black_, you should do it like this:
...
+```
+- Set a keyboard shortcut if you like, Ex. `ctrl-B`
+- Save: `Nothing`
```
Comment (ichard26) [inline docs/editor_integration.md:253]: I really hope that this doesn't turn into a source of "I JUST LOST A BUNCH OF MY WORK AND I BLAME YOU" type of bug reports, but I don't use gedit so I have no idea.

**Review by ichard26 — CHANGES_REQUESTED** [review]
Please run `pre-commit run prettier -a` locally, stage, commit (or amend), and push since the Prettier pre-commit hook is failing right now. I also have a few concerns about the configuration you've provided.

Thanks for submitting an integrations PR! It really helps out the project. Feel free to add yourself to the Authors list in the README.

File: docs/editor_integration.md (hunk 1 above)
Comment (Architrixs) [reply docs/editor_integration.md]: Alright, thanks for looking into this..
fixed the variable thing... find that I put a space between 'Name' and '=', in the docs while formatting.
Again added a pull request,

**Pull request closed without merging**